### PR TITLE
✨ Update IPAM image reference

### PR DESCRIPTION
**Pull request by furkatgofurov7**
**What this PR does / why we need it**:

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

### CODE REVIEW

File: config/ipam/image_patch.yaml [hunk 1]
```diff
@@ -8,5 +8,5 @@ spec:
     spec:
       containers:
       # Change the value of image field below to your controller image URL
-      - image: quay.io/metal3-io/ip-address-manager:v0.1.1
+      - image: quay.io/metal3-io/ip-address-manager:latest
```
Comment (fmuyassarov) [inline config/ipam/image_patch.yaml:11]: what is that for?

Comment (fmuyassarov) [reply config/ipam/image_patch.yaml:11]: isn't it better to pin the version?

Comment (furkatgofurov7) [reply config/ipam/image_patch.yaml:11]: we do once we have image in quay with release version tag pinned, for now we do not have release with v1beta1 API changes so we use latest(once we release ipam v1.0.X from main branch, we should have quay image:  quay.io/metal3-io/ip-address-manager:v1.0.0 which we will reference from here)

Comment (fmuyassarov) [reply config/ipam/image_patch.yaml:11]: I see. Makes sense
/lgtm

**Review by kashifest — APPROVED** [review]

**Pull request merged**